Naval update, is pretty big

SwedishNorwegian opened the pull request:
- Adjusted the shiplayouts of all Battleship hulls.
- All Battleship Hulls have the capability of running 4 Primary Batteries, increasing tech increases the amount of Secondaries / AA you can mount
- Added Twin and Quadruple Turret layouts for the ability to do more historical designs.
- Added Twin Turrets for Cruisers, Light and Heavy
- All starting navies have been adjusted to a more historically appropriate layout. Though some designs will not be 100% accurate due to game limitations.

SwedishNorwegian committed:
Swedish Tanks

SwedishNorwegian committed:
d

SwedishNorwegian committed:
Light Tanks done, now Arty, AA and TD

SwedishNorwegian committed:
Tank Destroyers

SwedishNorwegian committed:
Anti-Air

SwedishNorwegian committed:
GFX Tanks

SwedishNorwegian committed:
boom

AbbusF committed:
Most Generic Stuff

SwedishNorwegian committed:
Swedish historical planes sorted.

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
Icons mostly fixed for SWE air

SwedishNorwegian committed:
Fighters done

SwedishNorwegian committed:
s

SwedishNorwegian committed:
Update SWE_planes.txt

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
Ship updates to NOR, DEN, SWE

SwedishNorwegian committed:
FIN CD done

SwedishNorwegian committed:
South America Done

DutchPlanDerLindeSus committed:
Yugo Navy

DutchPlanDerLindeSus committed:
Merge branch 'historical-designs' of https://github.com/deliciousmods/1956_beta into historical-designs

SwedishNorwegian committed:
US Battleships corrected.

SwedishNorwegian committed:
Japanese Battleships and Battlecruisers

SwedishNorwegian committed:
Fra

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
nölkjgsfdlöjmnkgfsd

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

DutchPlanDerLindeSus committed:
added historical production for Yugo

SwedishNorwegian committed:
Sorted Destroyers in proper order and fixed typo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into historical-designs

SwedishNorwegian committed:
a lot of gfx

SwedishNorwegian committed:
Loc. and tech added

SwedishNorwegian committed:
mroe

SwedishNorwegian committed:
German Cruisers updated

SwedishNorwegian committed:
gui¨'

SwedishNorwegian committed:
ENG done

SwedishNorwegian committed:
French Ships

SwedishNorwegian committed:
Update FRA_1936_naval_mtg.txt

SwedishNorwegian committed:
JAP

SwedishNorwegian committed:
Merge branch 'historical-designs' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Update ship_hull_heavy.txt

SwedishNorwegian committed:
Proper coooking

SwedishNorwegian committed:
FRA, USA, ENG Navies updated AGAIN

SwedishNorwegian committed:
GER done

SwedishNorwegian committed:
Italian BB, BC CA done

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
,.nkbmfsda

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
fasdfsadfads

SwedishNorwegian committed:
sgdfgfsd

SwedishNorwegian committed:
gfagaga

SwedishNorwegian committed:
Revert "gfagaga"

This reverts commit a7aa16a0306690b01ce20d8fb0c6db9f5588d187.

SwedishNorwegian committed:
gafdsgasfd

SwedishNorwegian committed:
Update BAY - Bavaria.txt

SwedishNorwegian committed:
Update TUR - Turkey.txt

SwedishNorwegian committed:
Spain navy done

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
ARG, BRA, CAT, CHL, CNR, GLC and RAP ships adjusted

SwedishNorwegian committed:
fuckup amended in tur

SwedishNorwegian committed:
ITA and JAP fixed AGAIN

SwedishNorwegian committed:
Error log cleanup, AI Designs for BC/BB

SwedishNorwegian committed:
s

SwedishNorwegian committed:
Update AST - Australia.txt

SwedishNorwegian committed:
AST navy done

SwedishNorwegian committed:
CHI Navy

SwedishNorwegian committed:
Balanced?

SwedishNorwegian committed:
d

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Update 00_ship_modules.txt

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Update 00_r56_ship_modules.txt

SwedishNorwegian committed:
Damage and Speed

SwedishNorwegian committed:
Update RAP - Rapa Nui.txt

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

SwedishNorwegian committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

theodorhansson commented:
The Swedish AI will be blocked from building fighters of tier "improved" and above if the AI has yet to research heavy machine guns (or cannons for jets).

I suggest you move the changes to planes and tanks to a separate branch and fix the issues there, to keep this pull naval only.

theodorhansson commented on common/ai_equipment/SWE_planes.txt:
Possible deadlock

marijn211 committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

LemuelSora committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

LemuelSora committed:
Fixed Uruguay starting Navy

Set the proper armaments for Uruguays starting boat.
Historical data shows the ship was build in germany and armed with guns from Krupp, Skoda and Armstrong.
2x 120/45 Krupp QF, which exist word for word in game as Light Battery 1
4x 76/40 EOC N, which is a 76mmx40 gun from Elswick Ordnance a part of Armstrong Whitworth.
6x 37/40 Vickers
No good way to represent those, which means they get a basic AA to show a simple gun.
2x 450mm Torpedo Tube, which is also word for word the same as Torpedo Tube 1
It had a coal powered engine, which had 6k horsepowers, but the weakest i can give it is 35k.
Remaining data is inconsequential to the game.

Now Baris can leave me alone with this one thing...

marijn211 committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

marijn211 committed:
Merge branch 'Beta-Edits' into Naval-Gunnery-Update-Letsgooooo

marijn211 commented:
Temporarily marked draft, so no one merges this accidentally without sufficient checking, as this affects quite a few files
